src/helpers/helpers.rtl.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+var getRtlAdapter = function(rectX, width) {
+	return {
+		x: function(x) {
+			return rectX + rectX + width - x;
+		},
+		setWidth: function(w) {
+			width = w;
+		},
+		textAlign: function(align) {
+			if (align === 'center') {
+				return align;
+			}
+			return align === 'right' ? 'left' : 'right';
+		},
+		xPlus: function(x, value) {
+			return x - value;
+		},
+		leftForLtr: function(x, itemWidth) {
+			return x - itemWidth;
+		},
+	};
+};
+
+var getLtrAdapter = function() {
+	return {
+		x: function(x) {
+			return x;
+		},
+		setWidth: function(w) { // eslint-disable-line no-unused-vars
+		},
+		textAlign: function(align) {
+			return align;
+		},
+		xPlus: function(x, value) {
+			return x + value;
+		},
+		leftForLtr: function(x, _itemWidth) { // eslint-disable-line no-unused-vars
+			return x;
+		},
+	};
+};
+
+var getAdapter = function(rtl, rectX, width) {
+	if (rtl) {
+		return getRtlAdapter(rectX, width);
+	}
+
+	return getLtrAdapter();
+};
+
+var overrideTextDirection = function(ctx, direction) {
+	delete ctx.prevTextDirection;
+
+	if (direction === 'ltr' || direction === 'rtl') {
+		var original = [
+			ctx.canvas.style.getPropertyValue('direction'),
+			ctx.canvas.style.getPropertyPriority('direction'),
+		];
+
+		ctx.canvas.style.setProperty('direction', direction, 'important');
+		ctx.prevTextDirection = original;
+	}
+};
+
+var restoreTextDirection = function(ctx) {
```

Comment:
I'd change:
```js
var restoreTextDirection = function(ctx) {
	var original = ctx.prevTextDirection;
	if (original) {
		delete ctx.prevTextDirection;
		ctx.canvas.style.setProperty.apply(this, original);
	}
};
```

Reply:
Taken into consideration, with exact test against `undefined` which will save the runtime a lot of work with checking against a bunch of truthy types/values that we don't care about.